src/indexer1.rs

```diff
@@ -78,6 +78,7 @@ pub struct Indexer<S: LogStorage, P: Processor<S::Transaction>> {
     provider: RootProvider<Http<Client>>,
     ws_provider: Option<RootProvider<PubSubFrontend>>,
     fetch_interval: Duration,
+    block_range_limit: u64,
```

Comment:
Should be ```Option<u64>``` here, as there are lots of cases where you want to ignore this limit, moreover a lot of PRC's allow to fetch for any range and only limit the outcoming traffic

Reply:
Done